packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java

```diff
@@ -162,6 +162,16 @@ public static RestRuleModel createVariousActions()
         return ruleModel;
     }
 
+    public static RestRuleModel createRuleWithPrivateAction()
+    {
+        RestActionBodyExecTemplateModel mailAction = new RestActionBodyExecTemplateModel();
+        mailAction.setActionDefinitionId("mail");
```

Comment:
I would consider either using values from AccessRestrictionUtil constants, or even refactoring AccessRestrictionUtil for your needs here so that action restriction logic could be kept in one place. Like calling createMailParameters. With your changes there are 2 Rule services coexisting and there seems to be logic in AccessRestrictionUtil for one service and for RulesTestsUtils in another, I don't know when/if the first one will be removed but might be worth to keep it in mind for cleanup.

Reply:
I think it makes sense to have the two helpers - one is for action restriction, the other is for v1 rule creation. They have some overlap, and in my latest change I've removed some of the duplication.